doc/classes/Node.xml

```diff
@@ -63,7 +63,8 @@
 			</argument>
 			<description>
 				Called during the processing step of the main loop. Processing happens at every frame and as fast as possible, so the [code]delta[/code] time since the previous frame is not constant.
-				It is only called if processing has been enabled with [method set_process].
+				It is only called if processing is enabled, which is done automatically if this method is overriden,
+				and can be toggled with [method set_process].
```

Comment:
Please don't use a line break here, as 1 line == 1 paragraph in the processed output.